app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
+
```

Comment:
- [ ] With the given [_ERD_](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), whiles establishing a _has many_ **association** for the _User_ model, we need to consider that _Author_ is aliasing _User_ model and therefore  _author_id_ should be referenced as **foreign key** in _Post_, _Comment,_ and _Like_ models. Kindly ensure to improve upon this aspect, and our code should be similar to:
```ruby
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```